notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        batch_predict_starting_replica_count: int = 5,
</pre><p>I think we can remove this for simplicity</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>removed.</p>